snowflake/tests/snowflake_connector_patch/_snowflake_connector_patch/connector.py

```diff
@@ -58,5 +59,14 @@ def execute(self, query):
     def fetchall(self):
         return self.__data
```

Comment:
Do we need to keep this one now we use `fetchone`?

Reply:
Fair point, I think I was keeping it around "just in case" but that's not a good justification, I'll drop that.